error-handling/erros.js

```diff
@@ -1,4 +1,20 @@
-exports.handle500s = (err, req, res, next) => {
+exports.handle404s = (err, req, res, next) => {
   console.log(err);
-  res.status(500).send({ msg: "Internal Server Error" });
+  res.status(404).send({ msg: "Not Found" });
+};
+
+exports.idDoesNotExistError = (err, req, res, next) => {
```

Comment:
a bit picky but I'm not the biggest fan of the name of this function, what about handleCustomErrors?